SwC_python_sessions_ipynb/SwC_python_session-2-2.ipynb

```diff
@@ -117,11 +117,11 @@
     "\n",
     "Like functions, which are usable parts of a program, packages (also known as libraries) are reusable programs with several modules.\n",
     "\n",
-    "Many powerful tools are built into Python which can be imported without rewriting (or even reading) them in the current program.\n",
+    "There are many powerful tools built into Python, which can be imported without rewriting (or even reading) them in the current program.\n",
```

Comment:
I'm not sure exactly what you mean by 'rewriting' and 'reading' in this sentence - are you referring to the built-in functions like `enumerate`, `sort` etc? Or standard library modules?